initrd/bin/root-hashes-gui.sh

```diff
@@ -156,71 +158,268 @@ check_root_checksums() {
     fi
   fi
 }
+
+# Check if a device is an LVM2 PV, and if so print the VG name
+find_lvm_vg_name() {
+  TRACE_FUNC
+  local DEVICE VG
+  DEVICE="$1"
+
+  mkdir -p /tmp/root-hashes-gui
+  if ! lvm pvs "$DEVICE" >/tmp/root-hashes-gui/lvm_vg 2>/dev/null; then
+    DEBUG "Did not detect an LVM2 PV: $DEVICE"
```

Comment:
@JonathonHall-Purism As per output in my test, I get an error here: 
"DEBUG: Did not detect an LVM2 PV: /dev/mapper/qubes-dom0-root"

Reply:
That's not an error, that's normal since qubes-dom0-root is not an LVM2 PV.  It's a little confusing though, maybe I will just remove this trace

Reply:
(I added it thinking "if an LVM PV fails to open, I want to know whether it thought it was not a PV" but I think that'd be clear anyway)